easybuild/easyconfigs/p/PROCSE/PROCSE-2.0-goolf-1.4.10.eb

```diff
@@ -0,0 +1,39 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock='MakeCp'
+
+name = 'PROCSE'
+version = '2.0'
+
+homepage = 'http://www.biozentrum.unibas.ch/~nimwegen/cgi-bin/procse.cgi'
+description = """PROCSE (PRObabilistic Clustering of Sequences) is a Bayesian algorithm 
+ for clustering sequences over a finite alphabet (designed specifically for DNA sequences) 
+ into groups using a mixture model that assumes all sequences in a group derive from a common 
+ position-specific weight matrix (which is automatically integrated out of the likelihood). 
+ The program uses Markov chain Monte-Carlo sampling to sample the space o all partitions of 
+ the input sequences into subsets. Optionally, the user can specify informative priors regarding 
+ the expected numbers and sizes of clusters. The algorithm has been predominantly used to infer 
+ groups of binding sites forDNA or RNA binding proteins. 
+ Publication: http://www.pnas.org/content/99/11/7323.abstract """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://www.biozentrum.unibas.ch/~nimwegen/procse/']
+sources = ['%(namelower)s%(version)s.tar.gz']
+
+parallel = 1
+
+files_to_copy = [
+                (["procse"], "bin"),
+                "minimal_params", "README", "params_plus_explanation"
+] 
```

Comment:
4-space indentation.

this PR has only style remarks and is ready for test/merge